fix: join command failing on null values with --prefix-components-with-info-prop option (#1444)
Redocly/redocly-cli: .changeset/spotty-pets-doubt.md

---
"@redocly/cli": patch
---

> Fixed an issue when the `join` command was failing to process schemas containing `null` values with the `--prefix-components-with-info-prop` option.'

Reviewer: ```suggestion
Fixed a problem where the `join` command did not process schemas containing `null` values when the `--prefix-components-with-info-prop` option was used.'
```
